### PR TITLE
Add 2.8.0 release notes

### PR DESCRIPTION
**Pull request by svenklemm**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -38,87 +38,88 @@ If you have questions about distributed hypertables, join our #multinode channel
 on [community Slack](https://slack.timescale.com/) for installation details and
 follow these [setup instructions][distributed-hypertables-setup].
 
-### What's new in TimescaleDB 2.7:
+### What's new in TimescaleDB 2.8:
 
+* time_bucket now supports bucketing by month, year and timezone
+* 1 step CAgg policy management
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:44]:
```suggestion
* 1 step continuous aggregate policy management
```

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -38,87 +38,88 @@ If you have questions about distributed hypertables, join our #multinode channel
...
+* Migrate Continuous Aggregates to the new format
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:45]:
```suggestion
* Migrate continuous aggregates to the new format
```

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -38,87 +38,88 @@ If you have questions about distributed hypertables, join our #multinode channel
...
-chunks to the new data node for cluster rebalancing.
+## Release notes
 
+## 2.8.0 (2022-08-30)
 
-We're committed to developing these experiments, giving the community a chance to
-provide early feedback and influence the direction of TimescaleDB's development.
- We'll travel faster with your input!
+This release adds major new features since the 2.7.2 release.
+We deem it moderate priority for upgrading.
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:52]: We used to have standard statements for this in the comments, so you could just uncomment the one you needed. What's happened to that?

File: timescaledb/overview/release-notes/index.md [hunk 4]
```diff
@@ -38,87 +38,88 @@ If you have questions about distributed hypertables, join our #multinode channel
...
-`experimental-schema` label. Describe what you found, and tell us the steps or
-share the code snippet to recreate it.
+This release includes these noteworthy features:
 
-Several bugs fixed. See the [release notes](#release-notes) for more
-details.
+* time_bucket now supports bucketing by month, year and timezone
+* Improve performance of bulk SELECT and COPY for distributed hypertables
+* 1 step CAgg policy management
+* Migrate Continuous Aggregates to the new format
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:59]:
```suggestion
* Migrate continuous aggregates to the new format
```

File: timescaledb/overview/release-notes/index.md (hunk 3 above)
Comment (svenklemm) [reply timescaledb/overview/release-notes/index.md:52]: Sorry i'm not sure what you are referring to all this content is coming from https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md

**Review by charislam — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**